jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/TrackData.java

```diff
@@ -165,19 +165,30 @@ public void update() {
             }
         }
 
-        checkTimesConsistantcy();
+        checkTimesConsistency();
 
         length = times[times.length - 1];
     }
 
-    public void checkTimesConsistantcy() {
+    /**
+     * Verify that the
+     * {@link #times}, {@link #translations}, {@link #rotations}, and
+     * {@link #scales} vectors all have the same length, if present.
+     *
+     * @throws AssetLoadException if the lengths differ
+     */
+    public void checkTimesConsistency() {
         if ((translations != null && times.length != translations.length)
                 || (rotations != null && times.length != rotations.length)
                 || (scales != null && times.length != scales.length)) {
             throw new AssetLoadException("Inconsistent animation sampling ");
```

Comment:
space at the end of the error message is needed?

Reply:
can be removed

Reply:
However, I think removal shouldn't be part of this PR.